app/javascript/elm/src/Main.elm

```diff
@@ -997,8 +997,12 @@ viewNav navLogo sensors selectedSensorId page =
                             [ text "AirCasting" ]
                         , ul [ class "subnav-list" ]
                             [ li [ class "subnav-list__element" ]
+                                [ a [ class "subnav-list__link", href "/mobile_map" ]
```

Comment:
Please use `/map`. See the [other PR](https://github.com/HabitatMap/HabitatMap/pull/123/files) for more info.